protocols/kad/src/behaviour/test.rs

```diff
@@ -801,7 +803,6 @@ fn get_record() {
 
 #[test]
 fn get_record_many() {
-    // TODO: Randomise
```

Comment:
Why should we no longer randomise this value?

Reply:
readded